### ISSUE TITLE
Pan by dragging the graph

### ISSUE THREAD

**Issue by wenmin92** [npmgraph/npmgraph#79]
At present, for large graph, we can only use the scroll bar to view the complete graph, but it is not very convenient. It would be better to move the canvas directly.

**Comment by broofa** [comment]
Yup, agreed.  This feature needs to not interfere with other click-actions in the graph, however, which may complicate the implementation somewhat.  E.g. clicking on a node to view it in the inspector.

**Comment by wenmin92** [comment]
1. only click & hold white-space to drag
2. with modifier key, like shift or ctrl
3. or right click & hold to drag

**Comment by fregante** [comment]
If this is ever implemented, I think it would be nice to also implement a map-like interface for scrolling and zooming. Navigating large trees is rather limited and I find myself using Safari's pinch to zoom instead.
